components/ArticleCard.js

```diff
@@ -1,6 +1,15 @@
 import styled from "styled-components";
 import Image from "next/image";
 
-export default function ArticleCard({ imageUrl }) {
-  return <Image src={imageUrl} width={600} height={400}></Image>;
+export default function ArticleCard({ text, showContent }) {
+  return <StyledCard onClick={showContent}>{text}</StyledCard>;
 }
+
+const StyledCard = styled.section`
+  width: 400px;
+  height: 400px;
```

Comment:
If you use `px` then your layout is not responsive. You should change this and adapt the layout of your card to different screen sizes.

Reply:
thank you!